tests/foreman/ui/test_computeresource.py

```diff
@@ -47,7 +47,7 @@ def rhev_data():
 
 
 @pytest.mark.tier2
-def test_positive_end_to_end(session, rhev_data, module_org, module_location, module_target_sat):
```

Comment:
Instead of removing the use of module_org, module_location here, why not consider using it to verify CR creation into specific org and location rather than default org/location?

Reply:
Any changes to the code is not the scope of this PR. I would rather make this update in separate PR. I have removed only those fixtures which are not used in the test.

Reply:
yes, right, but you're modifying the existing usage of fixtures, which is also not in scope of the fixtures :) 
so I'd recommend either using module_org, module_location here instead of removing this fixture use OR we can remove this change from this PR itself to avoid code modification

Reply:
I am not modifying any usage of fixtures, I am just removing the unused fixtures from the test.